integrations/Vero/browser.js

```diff
@@ -37,17 +41,21 @@ class Vero {
    *
    * @param {Object} tags
    */
-  addOrRemoveTags(tags, userId) {
-    const addTags = get(tags, "add");
-    const removeTags = get(tags, "remove");
-    window._veroq.push([
-      "tags",
-      {
-        id: userId,
-        add: addTags,
-        remove: removeTags,
-      },
-    ]);
+  addOrRemoveTags(message) {
+    const tags = message.integrations?.[("vero", "Vero", "VERO")]?.tags;
+    if (isDefinedAndNotNull(tags)) {
+      const userId = message.userId || message.anonymousId;
+      const addTags = get(tags, "add") || [];
+      const removeTags = get(tags, "remove") || [];
```

Comment:
No need to use `get` here.

Reply:
@sauravlal-rudderstack just for clarity for any first level child we can directly refer from . notation
example 
```
tags.add
```
if `tags` is undefined and we do `tags.add` it will cause an unhandled exception
since we are already validating validity of tags due to if check we can use . notation 

if we try to reference a property of an undefined object js throws an exception, the `get` method actually handles that exception internally (making it less performant) so we have to use `get` when we actually need it